deepmd/infer/deep_pot.py

```diff
@@ -99,7 +100,9 @@ def eval(
         aparam: Optional[np.ndarray],
         mixed_type: bool,
         **kwargs: Any,
-    ) -> tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
+    ) -> tuple[
+        np.ndarray, np.ndarray, np.ndarray, np.ndarray, np.ndarray, Optional[np.ndarray]
+    ]:
```

Comment:
It should be

```suggestion
    ) -> Union[tuple[
        np.ndarray, np.ndarray, np.ndarray, np.ndarray, np.ndarray
    ], tuple[
        np.ndarray, np.ndarray, np.ndarray, np.ndarray, np.ndarray, np.ndarray
    ]]:
```

They have different meanings. `Optional[x]` is equivalent to `x | None`.